ontology/yaml/resources/HVAC/entity_types/FAN.yaml

```diff
@@ -472,6 +472,15 @@ FAN_EFHLC_COCDSP_NOCDSP:
   - COCDSP
   - NOCDSP
 
+FAN_FHLC_COCDSP_NOCDSP:
```

Comment:
Is this an exhaust fan? Can `FAN_EFHLC_COCDSP_NOCDSP` be used?

Reply:
Hello, this is a supply fan. I wanted to create a generic fan entity to be coherent with other entity type in DBO. For example, a supply fan or an exhaust fan use the same entity type HVAC/FAN_SS_VSC. The difference is in the name of the device that will be SF or EF. I wanted to extend that way of thinking for parking fan with this entity type. Is it make sense ?